RFC.rst

```diff
@@ -62,8 +62,13 @@ to fairly and consistently applying these principles to every aspect of
 managing this project. Project maintainers who do not follow or enforce
 the Code of Conduct may be permanently removed from the project team.
 
-This Code of Conduct applies both within project spaces and in public
-spaces when an individual is representing the project or its community.
+This Code of Conduct applies in all project spaces, as well as in other
+spaces where a user is acting in an official role as a spokesperson for
+the project.
+
+It does not apply to speech which does not pertain to be on behalf of the
+project (for example, opinions shared on social media which are not
+posted on an official project account).
```

Comment:
It's a really fuzzy line, which is part of the problem. On the one hand, "as soon as you're not on school grounds / on our mailing list anything is fair game" is clearly defining exactly how to dodge the rules and engage in abuse.  Taken the other direction, though, and we're back to Morality Clauses from employment contracts.  "If you say anything impolitic about gender anywhere, anytime, we will come after you" is not what we want (I would hope), but is the fear of many who have been opposing a CoC in general.

Perhaps referring to PHP business rather than "spokesperson for the project"?  That is, were I to go off on someone on Twitter about an RFC, I am not speaking "for PHP" (ie, not a spokesperson for PHP), and it's not on a PHP project space, but it's very clearly about matters within PHP's scope of work.